functests/common.go

```diff
@@ -28,6 +29,21 @@ func WaitForPVCBound(k8sClient *kubernetes.Clientset, pvcName string, pvcNamespa
 	}, 200*time.Second, 1*time.Second).ShouldNot(gomega.HaveOccurred())
 }
 
+// WaitForJobSucceeded waits for a Job with a given name and namespace to succeed until 200 seconds
+func WaitForJobSucceeded(k8sClient *kubernetes.Clientset, jobName string, jobNamespace string) {
+	gomega.Eventually(func() error {
+		job, err := k8sClient.BatchV1().Jobs(jobNamespace).Get(jobName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		if job.Status.Succeeded > 0 {
+			return nil
+		}
+		return fmt.Errorf("Waiting on job %s/%s to succeed when it is currently %d", jobName, jobNamespace, job.Status.Succeeded)
+	}, 200*time.Second, 1*time.Second).ShouldNot(gomega.HaveOccurred())
```

Comment:
Should(Succeed()) would be a bit more readable here: ShouldNot(HaveOccurred()) scans better when you're referencing the error itself as a variable. ShouldNot(HaveOccurred() is definitely correct though.

Reply:
Acknowledged!